auspice/server/getDatasetHelpers.js

```diff
@@ -36,6 +36,12 @@ const splitPrefixIntoParts = (prefix) => {
   /* The first part of the prefix is an optional source name.  The rest is the
    * source path parts.
    */
+
+  const isGroup = prefixParts[0] === "groups";
+  if (isGroup) {
+    prefixParts.shift();
+  }
+
   const sourceName = sources.has(prefixParts[0])
     ? prefixParts.shift()
     : "core";
```

Comment:
As a reminder for what we talked about yesterday, the `isGroup` conditional and `sources.has` conditional need to be combined in order to "break" the old paths in the way you want. Happy to chat more today.